pkg/minikube/constants/constants.go

```diff
@@ -51,6 +51,15 @@ func GetMinipath() string {
 	return filepath.Join(os.Getenv(MinikubeHome), ".minikube")
 }
 
+// SupportedArchTag retuns the archtag for images
```

Comment:
How about naming this function just ArchTag?

Also, fix spelling on "returns".

Reply:
YES. I agree with you.  changed function name to ArchTag